### ISSUE TITLE
Cannot Login - Error: Not logged properly

### ISSUE THREAD

**Issue by timothymowens** [BenoitZugmeyer/addic7ed-cli#42]
Whenever I try to login using the same credentials I use for the addic7ed.com website, I get the following error:

`Error: Not logged properly`

Command I'm running, per instructions/help:

`addic7ed login`

I've verified that I'm entering my credentials in properly, yet logging in fails.  I've tried on 3 different machines using addic7ed-cli, same issue on all 3 machines.

**Comment by timothymowens** [comment]
For now, I just logged into the addic7ed.com website, then ran the following in the browser debugging console:

`alert(document.cookie.match(/PHPSESSID=[^;]+/));`

Then copied the session ID from the java alert into my `~/.config/addic7ed` file below `[session]`.

To test, I downloaded a sub and checked my profile on the website to see if it recognized the download. 
 Sure enough, it logged the download; so the session I added to my addic7ed-cli config works for now.

I'm not sure why addic7ed-cli isn't able to post to `/dologin.php` properly.  It looks like the correct form fields are being used in the post (`username`, `password`).

Does it have anything to do with the site using https?

**Comment by vBm** [comment]
They have added captcha on their login page (it's random) so maybe that's the issue.

**Comment by BenoitZugmeyer** [comment]
Indeed, it seems they have put a silent captcha. Looks like it'll be hard to bypass. Sorry guys, I don't think we'll be able to login anymore.

**Comment by timothymowens** [comment]
Perhaps you could leave the session functionality in place, remove the login/logout for now and provide instructions for manually capturing the session ID and adding it to the .config/addic7ed file.  That worked perfectly for me.

**Comment by BenoitZugmeyer** [comment]
Ah, yes. I'll think about it. I could do something like displaying the instructions, and then a prompt for the session id instead of a prompt for username/password.

**Comment by Abhinav1217** [comment]
Same issue, I can't login because of reCAPTCHA
```
abhinav@abhinav-sol ~/.config $ addic7ed login
User: Abhinav1217
Password: 
Error: Make sure to reCHAPTCHA Try again
```

For the time being, I tried the solution mention above for using `alert(document.cookie.match(/PHPSESSID=[^;]+/));`. Is there any permanent solution to this?

**Comment by JoelSjogren** [comment]
Here's a relevant thread:
http://www.sub-talk.net/topic/7806-api-for-software-usage
